src/resources/License/LicenseInterfaces.ts

```diff
@@ -25,3 +25,14 @@ export interface LicenseModel {
     expirationDate: number;
     type: string;
 }
+
+/**
+ * Simplified version of a License's LicenseConnectorModel
+ * Mostly used to list what's possible, compared to a license who will itself only list what's enabled
```

Comment:
Out of curiosity, if this is not tied to the license, why is it fronted in the license resource? Wouldn't it make more sense to front this call in the `organization` or `source` resource?

Reply:
It's tied to the license, maybe my explanation is misleading.

The call using this model returns the list of every possible sourcetypes that CAN be enabled on an organization, whereas the license contains the list of sourcetypes that ARE enabled.